bin/ch/ch.cpp

```diff
@@ -671,6 +671,10 @@ HRESULT ExecuteTest(const char* fileName)
         }
     }
 Error:
+    if( fileContents != nullptr)
+    {
+        free((char*)fileContents);
```

Comment:
Actually, virtually no instance of ExternalArrayBuffer in ch actually cleans up its memory properly... I'll roll that set of fixes in here.